lib/SPIRV/libSPIRV/SPIRVType.h

```diff
@@ -656,20 +665,59 @@ class SPIRVTypeStruct : public SPIRVType {
     MemberTypeIdVec.resize(WordCount - 2);
   }
 
+  // TODO: Should we attach operands of continued instructions as well?
   std::vector<SPIRVEntry *> getNonLiteralOperands() const override {
     std::vector<SPIRVEntry *> Operands(MemberTypeIdVec.size());
     for (size_t I = 0, E = MemberTypeIdVec.size(); I < E; ++I)
       Operands[I] = getEntry(MemberTypeIdVec[I]);
     return Operands;
   }
+  void addContinuedInstruction(ContinuedInstType Inst) {
+    ContinuedInstructions.push_back(Inst);
+  }
+
+  void encodeChildren(spv_ostream &O) const override {
```

Comment:
Where this method is called?

Reply:
It is called by `SPIRVEntry::encodeAll` member https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/09d5508a737fac6bd491b93840d64957dfa41f7f/lib/SPIRV/libSPIRV/SPIRVEntry.cpp#L189 .
Then `encodeAll` is called when some `SPIRVEntry` is written to a file https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/09d5508a737fac6bd491b93840d64957dfa41f7f/lib/SPIRV/libSPIRV/SPIRVEntry.cpp#L458 .